sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2129,6 +2139,8 @@ class SQLConf extends Serializable with Logging {
 
   def maxNestedViewDepth: Int = getConf(SQLConf.MAX_NESTED_VIEW_DEPTH)
 
+  def deterministicUdfFoldEnabled: Boolean = getConf(DETERMINISTIC_UDF_FOLD_ENABLED)
+
```

Comment:
Ditto. Let's move this after `def defaultV2Catalog` at line 2231.

Reply:
ok, I will rebase as that line you mention doesn't exist in this current pr branch.